src/software/replay/replay_reader.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include <experimental/filesystem>
+
+#include "software/proto/tbots_replay.pb.h"
+#include "software/proto/sensor_msg.pb.h"
+
+class ReplayReader {
+   public:
+    explicit ReplayReader(const std::string& _replay_dir);
+    SensorMsg getNextFrame();
```

Comment:
To help be explicit to consumers, I think `getNextMsg()` would be a more clear name.

Reply:
changed